Tasks/Karachev/Task2/Task2.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+
+d1 = {1:("час", "первого"), 2:("два", "второго"), 3:("три", "третьего"), 4:("четыре", "четвертого"),
```

Comment:
pay attention to a proper variable names, d1 and d2 make sense only in some test/example code, not in a real solution